Coop_Listing_Site/Coop_Listing_Site/Migrations/201606022037353_Nullable_Student_GPA.cs

```diff
@@ -0,0 +1,18 @@
+namespace Coop_Listing_Site.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class Nullable_Student_GPA : DbMigration
+    {
+        public override void Up()
+        {
+            AlterColumn("dbo.StudentInfo", "GPA", c => c.Double());
+        }
+        
+        public override void Down()
+        {
+            AlterColumn("dbo.StudentInfo", "GPA", c => c.Double(nullable: false));
+        }
+    }
+}
```

Comment:
@Chef-Code how did you do this without making the GPA property in the StudentInfo model nullable?